public/src/js/constants/defaults.js

```diff
@@ -209,10 +209,10 @@ export default {
     reauthTimeout:         60000,
 
     imageCdnDomain:        '.guim.co.uk',
-    imageCdnDomainExpr:    /^https?:\/\/(.*)\.guim\.co\.uk\//,
-    imgIXDomainExpr:       /^https?:\/\/i\.guim\.co\.uk\/img\/static\//,
+    imageCdnDomainExpr:    /^https:\/\/(.*)\.guim\.co\.uk\//,
```

Comment:
Instead of changing this regular expression I think it's better to automatically replace `http:` with `https:` in `validate-image`